learn/pkg/learn/question.go

```diff
@@ -265,21 +265,31 @@ func (m *QuestionModel) printTextAnswerHTML(buf *bytes.Buffer, withAnswersMarked
 func (m *QuestionModel) answerContent(withAnswersMarked bool) (string, error) {
 	if !withAnswersMarked {
 		if evySrc, ok := m.AnswerText.(*evySource); ok {
-			return evySrc.source, nil
+			return removeCommentTaggedLines(evySrc.source), nil
 		}
 		return m.AnswerText.RenderOutput(), nil
 	}
 	answer, err := m.Frontmatter.getAnswer(m.privateKey)
 	if err != nil {
 		return "", fmt.Errorf("%w: (%s)", err, m.Filename())
 	}
-	content := answer.Text
+	content := removeCommentTags(answer.Text)
 	if !strings.HasSuffix(content, "\n") {
 		content += "\n"
 	}
 	return content, nil
 }
 
+func removeCommentTaggedLines(s string) string {
+	lines := strings.Split(s, "\n")
+	for i, line := range lines {
+		if strings.HasSuffix(line, " //levy:blank") {
```

Comment:
would `hide` be better than `blank`? I'm really not sure what the purpose of this annotation is - the commit does not say why you want this.

Reply:
ok, after discussion in person: `hide` doesn't sit right for me as my CSS bent 
brain reads hide as don't show at all including new line. Here the annotation
is meant to mean: `turn it into a "fill-in-the-blank" type exercise`.

I've updated the commit message to better reflect this but left the 
annotation as is.